addons/udes_stock/models/stock_picking.py

```diff
@@ -2787,11 +2787,24 @@ def raise_insufficiency_error(moves):
             processed = Picking.browse()
             by_type = lambda x: x.picking_type_id == picking_type
 
+            # Get the number of pickings that are not in batches.
+            domain = base_domain[:]
+            domain.append(("batch_id", "=", False))
```

Comment:
```suggestion
            domain = expression.AND([domain, [("batch_id", "=", False)]])
```

Its better to get used to use expression.And or expression.Or when appending to an existing domain. If there is any Or operator in existing domain it may not work as expected in some cases.